src/services/timers.py

```diff
@@ -0,0 +1,72 @@
+
+import datetime
+import time
+import traceback
+from typing import Tuple
+from infra import logger, utils 
+from infra.context import Context
+utils.load_env()
```

Comment:
This should happen in the main entry point, src/run.py, not here.
I will handle it in a follow-up commit :)